### PR TITLE
LGA 2678 improve documentation for endpoints in email orchestration api

### PR DESCRIPTION
**Pull request by BenMillar-MOJ**
### [LGA 2696](https://dsdmoj.atlassian.net/jira/software/c/projects/LGA/boards/226?modal=detail&selectedIssue=LGA-2696)

Improves the endpoint documentation based on the discussion during the Dev Team meeting

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: routers/email.py [hunk 1]
```diff
@@ -2,14 +2,23 @@
...
 email_router = APIRouter()
 
 
-@email_router.post("/email", summary="Send email to GOV.UK Notify", description="Will queue your email to be sent to GOV.UK Notify,\
-                    if their service is down the email will be held in an encrypted queue until a response is recieved.\
-                    If after two weeks no response is recieved your email be deleted.", status_code=201)
+@email_router.post("/email", summary=EmailRouter.summary, description=EmailRouter.description, status_code=201)
 async def send_email(email_request: Email, background_tasks: BackgroundTasks):
+    """
+    API Endpoint for /email.
```
Comment (psweeting1) [inline routers/email.py]: Is the `/` meant to be there?

Comment (psweeting1) [reply routers/email.py]: Oh! it's the URL. I think the wording needs to be something like. 

`API Endpoint for email route`

File: routers/email.py [hunk 2]
```diff
@@ -2,14 +2,23 @@
...
+    Recieves an email object and loads it onto the FIFO Queue.
+    A 422 response is sent if the email_request object is found to be invalid by pydantic
+    A 201 response is sent if the email_request object is valid
+
+        Parameters:
+            email_request (Email): An email object from the /email endpoint.
+                                   Please see models/reqest_models/email for the definition
+    """
```
Comment (Pheonnexx) [inline routers/email.py:22]: type should be request_models.

Comment (BenMillar-MOJ) [reply routers/email.py:22]: Apologies for that, thank you very much for pointing that out! 😄

File: routers/email.py (hunk 1 above)
Comment (BenMillar-MOJ) [reply routers/email.py]: Oh yes, that reads much clearer. Thank you very much, I will fix that! 😄

**Review by psweeting1 — APPROVED** [review]
LGTM

**Review by said-moj — APPROVED** [review]

**Pull request merged**